include/event2/bufferevent.h

```diff
@@ -280,7 +280,8 @@ int bufferevent_socket_get_dns_error(struct bufferevent *bev);
 
   NOTE that only socket bufferevents support this function.
 
-  @param base an event_base returned by event_init()
+  @param base an event_base returned by event_base_new() or the deprecated
+         event_init()
```

Comment:
Let's just remove any mentions about `event_init`, it should have been deprecated 10 years ago (and even more)